lib/DDG/Spice/YachtSpecs.pm

```diff
@@ -0,0 +1,34 @@
+package DDG::Spice::YachtSpecs;
+# ABSTRACT: Returns information of Yacht 
+
+use DDG::Spice;
+
+
+spice is_cached => 1; 
+
+#General attributes for meta data
+name "YachtSpecs";
+source "http://yachtharbour.com";
+icon_url "http://yachtharbour.com/favicon.ico";
+description "Gives out yacht specifications and photo";
+primary_example_queries "yacht azzam", "lady moura yacht";
+category "facts";
+topics "special_interest","travel","trivia";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/YachtSpecs.pm";
+attribution web => ["http://YachtHarbour.com"],
+            twitter => "yachtharbour";
+
+#Connection to API
+spice to => 'http://yachtharbour.com/tools/api.php?name=$1';
+spice wrap_jsonp_callback => 1;
+
+#Triggers
+triggers startend => "yacht", "superyacht", "megayacht", "luxury yacht", "motor yacht";
+
+# Handle statement
+handle remainder => sub {
+    return $_ if $_;
```

Comment:
We should probably add some negative trigger words here so we don't make redundant API calls for queries like "yacht for sale", "yacht club".

Alternatively, we might want to verify that the remainder is a single word -- or do yacht names contain spaces, @elephantRunAway?